dockerfiles/base/scripts/base/library.sh

```diff
@@ -74,56 +74,6 @@ server_is_booted() {
   fi
 }
 
```

Comment:
AFAIk this method is called from https://github.com/eclipse/che/blob/b446d114250d6e4e7b23e57a045929d9b3f20a6d/dockerfiles/base/scripts/base/startup_01_init.sh#L283